src/dom.js

```diff
@@ -53,7 +53,7 @@ export class Js {
 
   prepareWithUrl(urls, whichUrl = 'printed') {
     const
-      urlKeys = Object.keys(urls),
+      urlKeys = Object.keys(urls).filter((key) => (['printed', 'raw'].indexOf(key) > -1)),
```

Comment:
I got quite excited when I learned about the `()` for anonymous functions by eslint forcing me into it...